src/csmake.pl

```diff
@@ -86,27 +99,26 @@
 	spy('ar', 'spy-ar');
 	spy('mv', 'spy-mv');
 	spy('install', 'spy-install');
-	system(("make", @ARGV));
+	system(("make", @MAKEARGS));
```

Comment:
Aren't you loosing the `@ARGV` arguments?

Reply:
We don't want to mess csmake's arguments with make's.

Reply:
Yes, but *csmake* also takes make arguments, no?

Reply:
We only use them in make and gcc calls where we now use `MAKEARGS`.